daemons/dss-visitation-sentinel/.chalice/config.json

```diff
@@ -0,0 +1,28 @@
+{
+  "app_name": "dss-visitation-sentinel",
+  "environment_variables": {
+    "GOOGLE_APPLICATION_CREDENTIALS": "/var/task/domovoilib/gcp-credentials.json"
+  },
+  "stages": {
+    "dev": {
+      "api_gateway_stage": "dev",
+      "environment_variables": {}
+    },
+    "stochastic": {
+      "api_gateway_stage": "stochastic",
+      "environment_variables": {
+        "DSS_DEPLOYMENT_STAGE": "stochastic",
+        "DSS_ES_ENDPOINT": "search-dss-index-stochastic-yoahuyrbtn2jqbzgaeqbnx673i.us-west-2.es.amazonaws.com",
+        "DSS_S3_BUCKET": "dss-stochastic",
+        "DSS_GS_BUCKET": "dss-stochastic",
+        "API_HOST": "dss.dev.data.humancellatlas.org",
+        "DSS_SUBSCRIPTION_AUTHORIZED_DOMAINS": "chanzuckerberg.com ucsc.edu broadinstitute.org ebi.ac.uk human-cell-atlas-travis-test.iam.gserviceaccount.com broad-dsde-mint-dev.iam.gserviceaccount.com broad-dsde-mint-staging.iam.gserviceaccount.com",
+        "DSS_NOTIFICATION_SENDER": "DSS Notifier <noreply@humancellatlas.org>",
+        "DSS_S3_CHECKOUT_BUCKET": "org-humancellatlas-dss-checkout-dev"
```

Comment:
All of these variables seem like they are set by the deployer, please remove them.

Reply:
fixed.